<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Input_Output_Properties</class>
 <widget class="QWidget" name="Input_Output_Properties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QSpinBox" name="inputBox"/>
   </item>
   <item row="2" column="5">
    <widget class="QSpinBox" name="outputBox"/>
   </item>
   <item row="0" column="5">
    <widget class="QSpinBox" name="IDBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5">
    <widget class="QSpinBox" name="valueBox"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="delButton">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
